.env.example

```
@@ -1,12 +1,20 @@
 # Database
 DATABASE_HOST=localhost
+DATABASE_PORT=25061
 DATABASE_USER=root
 DATABASE_PASSWORD=devpassword
 DATABASE_NAME=mpga
 
 # BetterAuth (admin app only)
 BETTER_AUTH_SECRET=change-this-in-production
 BETTER_AUTH_URL=http://localhost:26101
+NEXT_PUBLIC_BETTER_AUTH_URL=http://localhost:4100
+NEXT_PUBLIC_APP_URL=http://localhost:4100
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_APP_URL key should go before the NEXT_PUBLIC_BETTER_AUTH_URL key

(UnorderedKey)